### ISSUE TITLE
Inconsistency in answers for FoC 1 2021 and 2022

### ISSUE THREAD

**Issue by E1Bos** [ClassicConor/UoKCSYear1ExamNotes2024#1]
This issue involves question 3.a of the 2021 paper and 2.a of the 2022 paper.

Attached are the questions for convenience (2021 above, 2022 below):
![vivaldi_6T7k6JDzqu](https://github.com/ClassicConor/UoKCSYear1ExamNotes2024/assets/66886116/feaaa16d-17ee-4760-8805-716af8f3bf33)
![vivaldi_nObVc08k8M](https://github.com/ClassicConor/UoKCSYear1ExamNotes2024/assets/66886116/677185f8-8e49-4116-a09d-761810d5cb60)

In the official and unofficial answers for the 2021 paper, the first 4 terms are {2, 8, 18, 32}, as expected, since it's a summation. However, in the 2022 paper, the answers are {5, 25, 125} which is the term itself, excluding the sum of the previous terms.

Attached are the answers given on the slides for both papers (2021 above, 2022 below):
![vivaldi_KpCHVsp2m7](https://github.com/ClassicConor/UoKCSYear1ExamNotes2024/assets/66886116/0035a2c7-43c9-4374-9110-1d91ca43a20d)
![vivaldi_0bxrxXUzEN](https://github.com/ClassicConor/UoKCSYear1ExamNotes2024/assets/66886116/ad113f2f-1e54-4ac4-99a3-97f3326025ff)

Going off the logic of the 2021 paper, the 2022 answers should be {5, 30, 155}, and based on the 2022 paper's logic, the answers on the 2021 paper should be {2, 6, 10, 14}

The questions are worded the same way, however the generic solution to each problem is different.

The answers in the unofficial answer for the 2022 paper seem to have been overwritten following the release of the official paper, however, I suspect the original solution of {5, 30, 155} was correct and the math division made a mistake when releasing the official answers from the 2022 paper.

I've emailed someone in the math division of computing for clarification, however am yet to receive a reply.

**Comment by ClassicConor** [comment]
I may have added the notes incorrectly, as I simply took screenshots of the video, then attempted to attach them to one another. The "Official" versions of the answers are just screenshots. The chances are that their versions are actually correct, and I've just rushed them, so please don't email them.

I may have rushed it when I took the notes down, but also, let's be honest, this stuff is pretty easy. I'm going to remove the notes for FOC 1 after the exam anyway so that I can rework it (perhaps for the next academic year for them to find useful also).

Hope this helps

**Comment by ClassicConor** [comment]
TLDR: it's my mistake, as I rushed taking screenshots and didn't do it properly.

**EDIT - Disregard my previous response, and this response, as the issue was with the slide itself, and not my editing.**

**Comment by E1Bos** [comment]
I looked over the Panopto recording. **You did not make a mistake and did screenshot the slide properly.** One of the slides is wrong. I emailed one of the TAs in my sessions for clarification since this problem seems to repeat itself in every exam, would be good to know which answer is correct. The questions are easy, but clarification is good for a question worth 4 marks, losing out on free points sucks.

**Comment by ClassicConor** [comment]
Ah, okay, that's good to know. I've just double-checked on my end also, and it appears you're correct.

This reminder made me look at what I'd captured for the other lecture slides, and it turns out I was missing a couple of slides for one of the FOC 1's and the only current FOC 2. These are now, as far as I know, absolutely fixed, so thank you for making me look at what I'd already uploaded to GitHub.

**Comment by E1Bos** [comment]
There was no question about terms of a series on the exam so I guess that's problem solved. Still a bit weird that there's this discrepancy on the past papers.
